implementation/pubsub/rabbitmq/common/src/main/java/org/inaetics/dronessimulator/pubsub/rabbitmq/common/RabbitConnection.java

```diff
@@ -22,6 +22,12 @@
  * related to the connection to the broker.
  */
 public abstract class RabbitConnection {
+    /** The maximum number of connection attempts. */
+    private static final int MAX_CONNECTION_ATTEMPTS = 10;
+
+    /** The timeout between connection attempts in milliseconds. */
+    private static final long CONNECTION_ATTEMPT_TIMEOUT = 10000;
```

Comment:
Lower sleep time to 2 seconds en increase attempts to 40? This allows for a more responsive experience.